app/assets/stylesheets/partials/stamps.scss

```diff
@@ -1,4 +1,36 @@
+/*Global:*/
 .selected {
   background: #A333C833 !important;
   color: #A333C8 !important;
 }
+
+/*Label:*/
+.ui.progress {
+  background: white;
+}
+
+.second-bar {
+   z-index: 10;
+}
+
+/*Flag:*/
+.flag-column {
+  padding: 0 30px !important;
+}
+
+.new-flag-segment {
+  padding-bottom: 3rem !important;
+  margin-right: 29rem !important;
+}
+
+.ui.attached.label {
+  padding: 5px;
+}
+
+.battle-button {
+  margin: 0 !important;
+}
+
+.hidden-check {
```

Comment:
`display: none;` sollte in eine globale Klasse rein, wie zB `dnone` oder `hidden`, weil das immer wieder verwendet wird.